int21.c

```diff
@@ -155,6 +172,29 @@ void int21(uc_engine *uc)
                 break;
             }
 
+        case 0x0a: // buffered keyboard input
+            {
+                uint16_t r_dx, r_ds;
+                uint8_t max_buf = 0;
```

Comment:
there is no need to initialize value for "max_buf" here, as it will be overwritten later at line 184.
same for variables "buf" and "str" below.